website/views/options/settings/subscription.jade

```diff
@@ -50,14 +49,26 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template',
           .btn.btn-primary(ng-if='!user.purchased.plan.dateTerminated && user.purchased.plan.paymentMethod=="Stripe"', ng-click='Payments.showStripeEdit()')=env.t('subUpdateCard')
           .btn.btn-sm.btn-danger(ng-if='!user.purchased.plan.dateTerminated', ng-click='Payments.cancelSubscription()')=env.t('cancelSub')
 
-      .container-fluid.slight-vertical-padding(ng-if='!user.purchased.plan.customerId || (user.purchased.plan.customerId && user.purchased.plan.dateTerminated)')
-        small.muted=env.t('subscribeUsing')
-        .row.text-center
-          .col-xs-4
-            a.purchase.btn.btn-primary(ng-click='Payments.showStripe({subscription:_subscription.key, coupon:_subscription.coupon})', ng-disabled='!_subscription.key')= env.t('card')
-          .col-xs-4
-            a.purchase(href='/paypal/subscribe?_id={{user._id}}&apiToken={{User.settings.auth.apiToken}}&sub={{_subscription.key}}{{_subscription.coupon ? "&coupon="+_subscription.coupon : ""}}', ng-disabled='!_subscription.key')
-              img(src='https://www.paypalobjects.com/webstatic/en_US/i/buttons/pp-acceptance-small.png',alt=env.t('paypal'))
-          .col-xs-4
-            a.purchase(ng-click="Payments.amazonPayments.init({type: 'subscription', subscription:_subscription.key, coupon:_subscription.coupon})")
-              img(src='https://payments.amazon.com/gp/cba/button',alt=env.t('amazonPayments'))
+        .container-fluid.slight-vertical-padding(ng-if='!user.purchased.plan.customerId || (user.purchased.plan.customerId && user.purchased.plan.dateTerminated)')
+          small.muted=env.t('subscribeUsing')
+          .row.text-center
+            .col-xs-4
+              a.purchase.btn.btn-primary(ng-click='Payments.showStripe({subscription:_subscription.key, coupon:_subscription.coupon})', ng-disabled='!_subscription.key')= env.t('card')
+            .col-xs-4
+              a.purchase(href='/paypal/subscribe?_id={{user._id}}&apiToken={{User.settings.auth.apiToken}}&sub={{_subscription.key}}{{_subscription.coupon ? "&coupon="+_subscription.coupon : ""}}', ng-disabled='!_subscription.key')
+                img(src='https://www.paypalobjects.com/webstatic/en_US/i/buttons/pp-acceptance-small.png',alt=env.t('paypal'))
+            .col-xs-4
+              a.purchase(ng-click="Payments.amazonPayments.init({type: 'subscription', subscription:_subscription.key, coupon:_subscription.coupon})")
+                img(src='https://payments.amazon.com/gp/cba/button',alt=env.t('amazonPayments'))
+
+    h3= env.t('giftSubscription')
+    .row
+      .col-lg-12
+        ol
+          li
+            span= env.t('giftSubscriptionText1')
+          li
+            span= env.t('giftSubscriptionText2')
+          li
+            span= env.t('giftSubscriptionText3')
```

Comment:
I'm not sure those three span tags are needed. I think doing the list items like this would work:

`    li= env.t('giftSubscriptionText1')`

@mleah What do you think? Do the spans add anything to the layout?